src/test/java/core/basesyntax/validation/InputValidatorTest.java

```diff
@@ -0,0 +1,97 @@
+    private static InputValidator inputValidator;
+
+    @BeforeClass
+    public static void beforeAll() {
+        inputValidator = new InputValidatorImpl();
+    }
+
+    @Test
+    public void isValidInput_correctData_Ok() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+        Assert.assertTrue("Input data is incorrect!", true);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_incorrectArraySize_NotOk() {
+        String[] testData = new String[INCORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "apple";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_incorrectOperation_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "111111";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_emptyOperation_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_incorrectFruit_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "12314";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_emptyFruit_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_incorrectAmount_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "111111";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "amount";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_negativeAmount_NotOk() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "-10";
+        inputValidator.isValidInput(testData);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void isValidInput_emptyArrayIndexes_NotOk() {
```

Comment:
```suggestion
    public void isValidInput_emptyArray_NotOk() {
```
What means "emptyArrayIndexes" ?

Reply:
fixed